app/scripts/lib/router.js

```diff
@@ -103,6 +104,7 @@ define(function (require, exports, module) {
       'settings/display_name(/)': createChildViewHandler(DisplayNameView, SettingsView),
       'settings/emails(/)': createChildViewHandler(EmailsView, SettingsView),
       'signin(/)': createViewHandler(SignInView),
+      'signin_bounced(/)': createViewHandler(SignInBouncedView),
```

Comment:
We also need an entry in [server/lib/routes/get-frontend.js](https://github.com/mozilla/fxa-content-server/blob/f9f7f10d4310163ffc256d1536322d470dcfe646/server/lib/routes/get-frontend.js#L42)